src/CMakeLists.txt

```diff
@@ -17,12 +17,21 @@ if (GUI)
     if (WIN32)
         list (APPEND QBT_QT_COMPONENTS WinExtras)
     endif(WIN32)
+    if (APPLE)
+        list (APPEND QBT_GUI_OPTIONAL_LINK_LIBRARIES objc)
+        list (APPEND QBT_QT_COMPONENTS MacExtras)
+    endif (APPLE)
 endif (GUI)
 if (DBUS)
     list (APPEND QBT_QT_COMPONENTS DBus)
 endif (DBUS)
 find_package(Qt5 5.5.1 COMPONENTS ${QBT_QT_COMPONENTS} REQUIRED)
 
+if (GUI AND APPLE)
+    list (APPEND QBT_GUI_OPTIONAL_LINK_LIBRARIES ${Qt5MacExtras_LIBRARIES})
```

Comment:
Yes. `target_link_libraries(qbt_gui Qt5::MacExtras)` provides "-I", "-L", and "-l" compiler switches.